lms/services/group_info.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, _context, request):
         self._db = request.db
         self._lti_user = request.lti_user
 
-    def upsert(self, authority_provided_id, consumer_key, params):
+    def upsert(self, authority_provided_id, consumer_key, params, type_):
```

Comment:
```suggestion
    def upsert(self, h_group, consumer_key, params):
```

Now we have the value object, we could use it to simplify this call

Reply:
Done https://github.com/hypothesis/lms/pull/1717/commits/d6146b2fbe1f39bd89eafd29c1c1972f80c8d42b